djmoney/models/fields.py

```diff
@@ -34,12 +34,17 @@
 if 'djmoney_rates' in settings.INSTALLED_APPS:
     try:
         from djmoney_rates.utils import convert_money
-        AUTO_CONVERT_MONEY = True
+        RATES_INSTALLED = True
     except ImportError:
         # NOTE. djmoney_rates doesn't support Django 1.9+
-        AUTO_CONVERT_MONEY = False
+        RATES_INSTALLED = False
 else:
-    AUTO_CONVERT_MONEY = False
+    RATES_INSTALLED = False
+
+
+if getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False):
```

Comment:
I did it that way at first, but then you end up with a module level constant, and can't change it (easily) in the tests, at least, not with the normal 'settings' fixture provided by pytest-django - https://pytest-django.readthedocs.io/en/latest/helpers.html#settings - you would have to do some other kind of monkey patching of that constant. I thought this duplication was the lesser of two evils. What do you think?